### ISSUE TITLE
can't find org.ohnlp.medtagger.fit.Main

### ISSUE THREAD

**Issue by mingust** [OHNLP/MedTagger#22]
**Describe the bug**
I am not sure if this is a bug or not configure properly
it said: Error: Could not find or load main class org.ohnlp.medtagger.fit.Main

**Desktop (please complete the following information):**
 - OS: window 10
 - Java Version latest

**Additional context**

**Comment by mingust** [comment]
Oh, that’s from a command in run.bat file.

**Comment by qqndrew** [comment]
What is your execution pathway? Are you launching this through OHNLPTK/Backbone or are you running this directly?
Are you downloading the compiled release or are you running this from source?

**Comment by mingust** [comment]
Hi Andrew,

Oh...thanks replying.

I followed the instructions and downloaded the MedTagger-1.0.72 zip
After I unzip
The runmedtaggerwin.bat is under MedTagger-1.0.72\src\main\resources directory so I tried to run it there (since the bat file has put there)

I think after the unzip, that should be the source files and I am running from source.

One major disappointment is I can't find the covid19 directly for the rule files.

Thanks.

________________________________

**Comment by qqndrew** [comment]
Yeah you downloaded the source code, not the compiled program. You will need to compile yourself via maven or download the release binaries from MedTagger.zip at https://github.com/OHNLP/MedTagger/releases/tag/v1.0.72 

The COVID-19 project was consolidated to the pasc project, which is a union of both covid-19 signs/symptoms and signs/symptoms associated with long COVID/post-acute sequelae of COVID-19 (PASC)

**Comment by mingust** [comment]
Hi Andrew,

I checked each steps and had compiled medtagger using maven.
After compilation, a new medtagger-1.0.72.jar created
But when I run it an error occur;
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
        at org.ohnlp.medtagger.fit.Main.main(Main.java:22)

The java version I am using is 22.0.2

If you know what the problem is?  pls let me know, thanks.
________________________________
